modules/server/src/main/scala/observe/server/Systems.scala

```diff
@@ -71,10 +76,26 @@ object Systems {
     service:    CaService,
     tops:       Map[String, String]
   )(implicit L: Logger[IO], T: Temporal[IO]) {
-    def odbProxy[F[_]: Sync: Logger]: OdbProxy[F] = OdbProxy[F](
-      new Peer(settings.odb.renderString, 8443, null),
+    val reconnectionStrategy: WebSocketReconnectionStrategy =
+      (attempt, reason) =>
+        // Web Socket close codes: https://developer.mozilla.org/en-US/docs/Web/API/CloseEvent
+        if (reason.toOption.flatMap(_.toOption.flatMap(_.code)).exists(_ === 1000))
+          none
+        else // Increase the delay to get exponential backoff with a minimum of 1s and a max of 1m
+          FiniteDuration(
+            math.min(60.0, math.pow(2, attempt.toDouble - 1)).toLong,
+            TimeUnit.SECONDS
+          ).some
+
+    def odbProxy[F[_]: Async: Logger: WebSocketBackend]: F[OdbProxy[F]] = for {
+      sk <- ApolloWebSocketClient
```

Comment:
Are we listening to updates on the odb rather than polling?

Reply:
No, I went the easy way and reused the polling system in the original Seqexec